internal/backend/cron/google.go

```diff
@@ -2,20 +2,64 @@ package cron
 
 import (
 	"context"
+	"errors"
+	"time"
 
 	"go.temporal.io/sdk/workflow"
+	"go.uber.org/zap"
 
+	"go.autokitteh.dev/autokitteh/integrations/google/forms"
+	"go.autokitteh.dev/autokitteh/integrations/google/vars"
+	"go.autokitteh.dev/autokitteh/internal/backend/auth/authcontext"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 // TODO(INT-190): Move this to the Google integration package.
 
+type (
+	listActivity  func(context.Context) ([]sdktypes.ConnectionID, error)
+	renewActivity func(context.Context, sdktypes.ConnectionID) error
+)
+
+func (cr *Cron) renewGoogleEventWatchesWorkflow(wctx workflow.Context, la listActivity, ra renewActivity) error {
+	actx := temporalclient.WithActivityOptions(wctx, taskQueueName, cr.cfg.Activity)
+
+	var cids []sdktypes.ConnectionID
+	if err := workflow.ExecuteActivity(actx, la).Get(wctx, &cids); err != nil {
+		return err
+	}
+
+	errs := make([]error, 0)
```

Comment:
why the 0?

Reply:
you don't even need to make

```
var errs []error
```

append to nil works

Reply:
yes, but I need to declare `errs` in advance, and this is a good placeholder in case we want to optimize in the future, but today we don't, and that's why I'm using 0

Reply:
on second thought, your suggestion is still better, changing now